test cases/common/219 source set configuration_data/all.h

```diff
@@ -4,4 +4,4 @@ extern void h(void);
 extern void undefined(void);
 
 /* No extern here to get a common symbol */
-void (*p)(void);
+extern void (*p)(void);
```

Comment:
Comment on the line above needs updating somehow, since it's now in direct contradiction to the declaration.

Reply:
Oh, hurr, my bad.